src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -264,6 +264,27 @@ FabricInfo * RetrieveCurrentFabric(CommandHandler * aCommandHandler)
     return Server::GetInstance().GetFabricTable().FindFabricWithIndex(index);
 }
 
+CHIP_ERROR DeleteFabricFromTable(FabricIndex fabricIndex)
+{
+    ReturnLogErrorOnFailure(Server::GetInstance().GetFabricTable().Delete(fabricIndex));
+
+    // We need to withdraw the advertisement for the now-removed fabric, so need
+    // to restart advertising altogether.
+    app::DnssdServer::Instance().StartServer();
+
+    return CHIP_NO_ERROR;
+}
+
+void CleanupFabricContext(SessionManager * sessionMgr, FabricIndex fabricIndex)
```

Comment:
It does not seem like there can be a valid use for this if null session manager is passed.

I realize that from some contexts, we are storing pointers.  However, here it feels like we should either return a CHIP_ERROR if session manager is null, or change the parameter to a reference variable.  If we fail to pass a session manager, this method can't actually do its job.

```suggestion
void CleanupFabricContext(SessionManager & sessionMgr, FabricIndex fabricIndex)
```